### ISSUE TITLE
Staging Review finding: Tables must have a `table-title`

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#105699]
_Need help? Please review [how to read a Staging Review ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/anatomy-of-a-staging-review-issue-ticket). Tag `@platform-governance-team-members` on Slack if you need further assistance._

## Product Information
**Team:** Education Data Migration
**Product:** WEAMS Public
**Feature:** WEAMS Public

## Findings details
**VA.gov Experience Standard - issue:** [User can identify an element and its state.](https://design.va.gov/about/experience-standards/#user-can-identify-an-element-and-its-state)
**VA.gov Experience Standard - category:** [Usability](https://design.va.gov/about/experience-standards/#category-usability)
**Launch-blocking:** No
**Design System review:** No
**Collab Cycle Reviewer:** @briandeconinck (Accessibility)

### Description

When a certification test or other details page includes a table ([example](https://staging.va.gov/education/gi-bill-comparison-tool/licenses-certifications-and-prep-courses/results/2912@7/11G%20DATABASE%20CERTIFIED%20ASSOCIATE), you must include a brief description of what information is in the table using the `table-title` property on the `va-table` component.

When a screen reader user encounters a table, the expereince of navigating table cells is cumbersome and requires some cognitive effort. The `table-title` property maps to a `caption` element, which is announced to screen reader users as a summary of what the purpose of the table is before they have to dive into exploring the table cells. This gives a screen reader user the opportunity to decide "at a glance" if they want to explore the table or skip over it.

## Recommended action

Include the `table-title` prop with a brief description of the purpose of the table for each table used in the product.

## References

- **Accessibility Defect Severity:** [2: High. Should be fixed in 1-2 sprints post-launch.](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric#Accessibilitydefectseverityrubric-a11y-defect-2)
- **WCAG Success Criteria:** N/A
- **Modality:** Screen reader
- **Design System Component:** N/A
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** N/A
- **Content Style Guide:** N/A
- **Context:** N/A

---

### Next Steps for the VFS Team

_How did this touchpoint go? [Give feedback on the Collaboration Cycle](https://dj540s05.optimalworkshop.com/questions/f54355271969f160ef41182ba9ceb61c) at any time._
- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** If you believe that this issue is out of scope, not your team's responsibility, or a Design System issue, comment and tag `@platform-governance-team-members` on your team channel in Slack to provide an explanation and who you believe is responsible. The Governance team will follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.

**Comment by skulka07** [comment]
Hello @shiragoodman, @briandeconinck,

The table-title property when coded displays on the UI. Does what we have here look alright or does it need to be hidden?

![Image](https://github.com/user-attachments/assets/93876aba-f331-45b6-82f3-b6c1117d6141)

**Comment by briandeconinck** [comment]
What you've got is perfect, no need to have it hidden. Thank you!
